app/bundles/LeadBundle/Event/CompanyMergeEvent.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Mautic\LeadBundle\Event;
+
+use Mautic\LeadBundle\Entity\Company;
+use Symfony\Component\EventDispatcher\Event;
+
+/**
+ * Class CompanyMergeEvent.
+ */
+class CompanyMergeEvent extends Event
```

Comment:
I suggest to have value objects final. The comment has no value so I suggest to remove it.
```suggestion
final class CompanyMergeEvent extends Event
```